src/clojurians_log/views.clj

```diff
@@ -271,6 +276,18 @@
        (get-in context [:request :clojurians-log.application/title])]]
      (:data/about-hiccup context)]]])
 
+(defn- sitemap-xml [{:data/keys [channel-day-tuples] :as context}]
+  [:urlset {:xmlns "http://www.sitemaps.org/schemas/sitemap/0.9"}
+   (for [[{:channel/keys [name]} channel-days] channel-day-tuples]
+     (for [[day cnt] channel-days]
+       [:url
+        [:loc (str "https://clojurians-log.clojureverse.org"
```

Comment:
How can we avoid this hardcoded Url?

Reply:
There should be a `:data/http-origin` in the context map that you can use.